ui/src/utils/ajax.js

```diff
@@ -2,40 +2,41 @@ import axios from 'axios'
 
 let links
 
+// do not prefix route with basepath, ex. for external links
+const addBasepath = (url, excludeBasepath) => {
+  const basepath = window.basepath || ''
+
+  return excludeBasepath ? url : `${basepath}${url}`
+}
+
 const generateResponseWithLinks = (response, {auth, logout, external}) => ({
   ...response,
   auth: {links: auth},
   logoutLink: logout,
   external,
 })
 
-const AJAX = async ({
-  url,
-  resource,
-  id,
-  method = 'GET',
-  data = {},
-  params = {},
-  headers = {},
-}) => {
+const AJAX = async (
+  {url, resource, id, method = 'GET', data = {}, params = {}, headers = {}},
+  excludeBasepath
```

Comment:
Is the value of this truthy/falsy? if so, maybe we should refactor it to be a config object once we have more things we want to add as configs.